### ISSUE TITLE
Lower case links to notes imported from Tomboy

### ISSUE THREAD

**Issue by lydiaslo** [tomboy-notes/tomboy-ng#143]
Links to notes imported from Tomboy are converted to lower case.   So, e.g., the old note title is "Todo Today" and I type "Todo Today" into a new note and it is converted to "todo today" in the new note.  The note title in the original still has the original case.

**Comment by davidbannon** [comment]
Yes, it does do that. And there is a reason although I cannot remember exactly what that reason is/was right now.
I will have a look.
Davo

**Comment by davidbannon** [comment]
Hmm, Lydiaslo, can I check which version of tomboy-ng you are using please ?

My records indicate that I fixed this issue in version 0.25, around Christmas time.   From main menu, click 'About'.

If you are still seeing it in a current version, I'll have to have a bit of a think !

I note you mention that this happens with notes imported from Tomboy ?  Does it only happen with imported notes ?  (That would worry me even more !)

Davo

**Comment by lydiaslo** [comment]
Hi Davo -- I am using version 0.24b.  I'm running an older version of Ubuntu and I wasn't able to get the latest version to install -- but I will try again this weekend since it sounds like that fixes the problem!  Thank you!

**Comment by davidbannon** [comment]
Ah, that makes sense. "An older version" of Ubuntu ?  I suspect you are using 16.04 and being hit with the appmenu-gtk2-module bug ?  Sigh ...
The current short term release of Ubuntu does need that module and its listed as a dependency in the deb. Things like my 18.04 have it available (but does not need it) and 16.04 does not have it available. I am hoping that the next long term will have it mentioned in the GTK2 kit and I can drop it from the dependencies list. In the mean time, its a mess !
I am working on a new release right now am unsure of how to fix this messy problem. If I don't mention it in the deb, you will be able to install and users of U19.10 won't be able to run.....
An alternative for you is to use the tgz file and do your install manually, you must already have the (real) dependencies there if you can run 0.24 so its not too hard. But harder than using the deb.

**Comment by aguador** [comment]
Yes, this is a real pain, and an unnecessary one it would seem as I have the latest version running on the older version of MX Linux that does not have the appmenu package. I did not do the unspeakable, but something similar, which was to unpack the .deb and overwrite the main package on the system. So the disgrace as David knows, is that the program will run but the system blocks the install.

@lydiaslo: if you are in a position to upgrade, that is probably a good step all round. Life keeps getting in the way of my upgrade of MX Linux which is neither my daily driver nor does it have (because of some under the hood changes) a path like the Ubuntu LTS to LTS+1 system (i.e., that 16.04 can upgrade to 18.04 but not directly to 20.04 without the intermediary LTS). In your case, if you are on 16.04, the 18.04 upgrade is good as the version has been tested and refined -- and it positions you for the upgrade to 20.04 later.

**Comment by davidbannon** [comment]
Ah, but I have good news !

There is a beta of U20.04 available and I have tested against it. Not only does it not need appmenu but it also arrives, out of the box with almost every thing needed to run tomboy-ng. U19.10 needed a 200meg dependencies download (so I recommended the  Qt5 version, needing only 50meg).

That means, I believe, the next release of tomboy-ng will not have a dependency on appmenu and will install fine just about everywhere.  I will provide a note for U19.10 users saying they must manually install appmenu but U19.10 runs out in a few months anyway.
tomboy-ng 0.27 is in testing now ...

Davo
